docs/DOCKER-MACHINE.md

```diff
@@ -0,0 +1,45 @@
+# Docker Machine
+
+Before the Docker native client we used docker-machine. Here are the steps for this original setup process.
+
+## Docker dependencies
+
+The project is managed via Docker containers. Right now we're using Docker
+version 1.9.0 with Docker Machine version 0.5.0.
+
+Commands below assume OS X and preference to install libraries via Homebrew.
```

Comment:
Good thinking. @andydutch I'm making some modifications to the setup process in README.md. Thinking about killing the old setup and just replacing with essentially "Download Docker Native Client". Any thoughts about keeping the legacy setup process doc?